demo/vl-typography.html

```diff
@@ -170,6 +169,13 @@ <h6>Heading 6</h6>
                     </tbody>
                 </table>
             </vl-typography>
+        </vl-demo> -->
+
+        <vl-demo data-vl-title="Parameters">
+            <vl-typography data-vl-parameters='{"key1": "tempus", "key2": "ipsum"}'>
```

Comment:
De quotes wordt escaped:
![Screenshot 2020-07-24 at 21 01 58](https://user-images.githubusercontent.com/51441000/88426184-06049f00-cdf1-11ea-9730-06af8084c4f8.png)

Reply:
Dat is inderdaad een gekend probleem en zal opgelost worden in [#17](https://github.com/milieuinfo/webcomponent-vl-ui-demo/issues/17).